modules/apps/headless/headless-asset-library/headless-asset-library-impl/rest-openapi.yaml

```diff
@@ -696,6 +696,47 @@ paths:
                     description:
                         default response
             tags: ["AssetLibrary"]
+    "/asset-libraries/by-external-reference-code/{externalReferenceCode}/pins":
```

Comment:
A doubt: would it be better called in singular i.e. `.../pin` ?
It seems that we are always working on a single pin